settings/resource_enhanced_security_monitoring_setting.go

```diff
@@ -0,0 +1,51 @@
+package settings
+
+import (
+	"context"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/settings"
+)
+
+// Enhanced Security Monitoring setting
+var enhancedSecurityMonitoringFieldMask = strings.Join([]string{
+	"enhanced_security_monitoring_workspace.is_enabled",
+}, ",")
+var enhancedSecurityMonitoringSetting = workspaceSetting[settings.EnhancedSecurityMonitoringSetting]{
```

Comment:
my understanding is that this setting can be set at both account & workspace level - can we use `accountWorkspaceSetting` instead

Reply:
those are 2 separate settings with different fields:

1. Account level - https://github.com/databricks/universe/blob/ea7ffc3ed4b3ef62407dc7ca74e1210ab926e84a/settings-policy/api/proto/external/service.proto#L63
2. Workspace level - https://github.com/databricks/universe/blob/ea7ffc3ed4b3ef62407dc7ca74e1210ab926e84a/settings-policy/api/proto/external/service.proto#L66

Should they still be combined?

Reply:
this is interesting - so structure wise they are the same - <something>.<is_enabled>, but the top-level field name is different. From a usability perspective, it is easier for users to have a single resource, but let's see how much work it is to add aliasing in TF

Reply:
I don't think they are the same - the account level has the field`is_enforced` and workspace level is `is_enabled`. The account level is only used for setting defaults for new workspaces (new workspaces auto inherit the account level setting if set).